po/ko.po

```diff
@@ -20348,8 +20594,9 @@ msgstr ""
 
 #: src/exercises/concurrency/chat-app.md:93
 #: src/exercises/concurrency/solutions-afternoon.md:155
+#, fuzzy
 msgid "// Wrap the raw TCP stream into a websocket.\n"
-msgstr ""
+msgstr "/// 버퍼에 위젯을 그립니다.\n"
```

Comment:
this is a translation for "draw widget on the buffer"

Reply:
That's another fuzzy match, and it's going to be addresses in a PR for the Concurrency section.